### PR TITLE
cleanup autoloading

### PR DESCRIPTION
**Pull request by lsmith77**
sync with Doctrine DBAL https://github.com/jackalope/jackalope-doctrine-dbal/pull/288

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -8,14 +11,9 @@
...
-/** make sure we get ALL infos from php */
-error_reporting(E_ALL | E_STRICT);
-
-### Load classes needed for jackalope unit tests ###
-require 'Jackalope/Transport/Jackrabbit/JackrabbitTestCase.php';
-
 ### Load the implementation loader class ###
-require 'inc/ImplementationLoader.php';
+require_once __DIR__ . '/ImplementationLoader.php';
+require_once __DIR__ . '/JackrabbitFixtureLoader.php';
```
Comment (dbu) [inline tests/bootstrap.php]: are these not in the inc/ subfolder?

Comment (lsmith77) [inline tests/bootstrap.php]: not any more .. I moved them to the top level .. we do not habe an inc folder in Doctrine DBAL and imho inc folders are wierd :)

Comment (dbu) [inline tests/bootstrap.php]: right. should have looked at the file titles in the diff as well to see that you move them.

**Comment by dbu** [comment]
looks like we accept a too low version of jackalope/jackalope

**Pull request merged**